infoschema/tables.go

```diff
@@ -1997,6 +2011,81 @@ func dataForTiDBClusterInfo(ctx sessionctx.Context) ([][]types.Datum, error) {
 	return rows, nil
 }
 
+type addressWithType struct {
+	addr string
+	// TiDB/PD/TiKV
+	tp string
+}
+
+func dataForClusterLoadInfo() ([][]types.Datum, error) {
+	serversInfo, err := infosync.GetAllServerInfo(context.Background())
```

Comment:
Just retrieve load info from TiDB? It's better to retrieve all cluster component from `TiDB_CLUSTER_INFO`.

Reply:
Address, but after PD/TiKV implement the diagnostics grpc service.